opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -462,7 +462,7 @@ def _target():
         span_processor.shutdown()
 
     @mark.skipif(
-        python_implementation() == "PyPy" and system() == "Windows",
+        python_implementation() == "PyPy" or system() == "Windows",
         reason="This test randomly fails with huge delta in Windows with PyPy",
```

Comment:
Should update the comment to reflect "or"?

Reply:
Right, fixed